src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -970,7 +973,7 @@ protected boolean isCategoryEnabled(@NonNull SCMHeadCategory category) {
     public void afterSave() {
         SCMSourceOwner owner = getOwner();
         if (owner != null) {
-            GitHubWebHook.get().registerHookFor(owner);
+            // TODO Post JENKINS-39533 GitHubWebHook.get().registerHookFor(owner);
```

Comment:
Do you mean JENKINS-39355? And why are we waiting for something “post” that? I thought the whole point of this PR was to use those APIs.